docs/installation.md

```diff
@@ -288,20 +288,27 @@ export SM_DEV_DB_PORT=3306 # or 3307
 
 You'll want to set the following environment variables (permanently) in your local development environment.
 
-The `SM_ENVIRONMENT`, `SM_LOCALONLY_DEFAULTUSER` and `SM_ALLOWALLACCESS` environment variables allow access to a local metamist server without providing a bearer token.
+The `SM_ENVIRONMENT` and `SM_LOCALONLY_DEFAULTUSER` environment variables allow access to a local metamist server without providing a bearer token.
 
 This will allow you to test the front-end components that access data. This happens automatically on the production instance through the Google identity-aware-proxy.
 
 ```bash
 # ensures the SWAGGER page points to your local: (localhost:8000/docs)
 # and ensures if you use the PythonAPI, it also points to your local
 export SM_ENVIRONMENT=LOCAL
-# skips permission checks in your local environment
-export SM_ALLOWALLACCESS=true
 # uses your username as the "author" in requests
 export SM_LOCALONLY_DEFAULTUSER=$(whoami)
 ```
 
+To allow the sytem to be bootstrapped and create the initial project, you'll need to add yourself to the two admin groups that allow creating projects and updating project members:
+
+```sql
+INSERT INTO group_member(group_id, member)
```

Comment:
Potentially give the mariadb --execute "..." command with the whoami substitution?

Reply:
I think there's maybe too many permutations and combinations of possible setups to provide an exact command? Personally I don't have the mariadb executable available because I run it in docker, so would require a docker command, but that wouldn't work for others.